datafeeds/parsers/fms_api/fms_api_team_avatar_parser.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from google.appengine.ext import ndb
+
+from models.media import Media
+from models.team import Team
+from consts.media_type import MediaType
+from helpers.media_manipulator import MediaManipulator
+
+
+class FMSAPITeamAvatarParser(object):
+    def __init__(self, year,  short=None):
+        self.year = year
+        self.event_short = short
+
+    def parse(self, response):
+        """
+        Parse team avatar from FMSAPI
+        """
+
+        # Get team avatar
+        current_page = response['pageCurrent']
+        total_pages = response['pageTotal']
+        teams = response['teams']
+        avatars = []
+
+        for teamData in teams:
+            if not teamData['encodedAvatar']:
+                encoded_avatar = None
```

Comment:
This should be `continue`, so we don't create any avatar instead of creating blank ones.